lib/rubocop/cop/rspec/empty_example_group.rb

```diff
@@ -60,17 +60,50 @@ module RSpec
       class EmptyExampleGroup < Cop
         MSG = 'Empty example group detected.'
 
-        def_node_search :contains_example?, <<-PATTERN
+        def_node_matcher :example_group_body, <<~PATTERN
+          (block #{ExampleGroups::ALL.send_pattern} args $_)
+        PATTERN
```

Comment:
Other cops' specs break if I change `example_group?` to yield its body.
I decided to copy this pattern over here rather than defining it as
```
$(block #{ExampleGroups::ALL.send_pattern} args $_)
```
for every cop that would ignore the second argument. But it might be possible, and I don't have a preference here.